sphinx/builders/linkcheck.py

```diff
@@ -125,10 +127,18 @@ def check_uri() -> Tuple[str, str, int]:
             except UnicodeError:
                 req_url = encode_uri(req_url)
 
+            # Get auth info, if any
+            for rex, auth_info in self.auth:
```

Comment:
bikeshed: I feel `pattern` or `regexp` is good for this. `rex` is a bit ambiguous to me.

Reply:
Agree, `rex` is used in other places of the file to name essentially the same type of thing